.github/workflows/push_trigger.yml

```diff
@@ -65,14 +65,14 @@ jobs:
         name: release
         path: ./release.zip
 
-    - name: Build Stage-Group-1 image
+    - name: Build init-stage-group image
```

Comment:
Lets not change the image name, lets keep it as group-1, group-2 etc, since we will soon change the groups

Reply:
Let follow this everywhere below